mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/UserScreen.kt

```diff
@@ -22,6 +17,7 @@ import ch.epfl.sdp.mobile.ui.social.*
  * @param profileBar part of slot construct that represents the tab bar.
  * @param matches the part that is responsible for the list of all matches
  * @param lazyColumnState to keep state of LazyColumn
+ *

```

Comment:
I think you meant to add some doc here but forgot ^^

```suggestion
 * onMatchClick a callback called when a [ChessMatch] is clicked
```